packages/blockchain-wallet-v4-frontend/src/scenes/app.tsx

```diff
@@ -117,6 +121,11 @@ const App = ({
                       <Switch>
                         {/* Unauthenticated Wallet routes */}
                         <Route path='/app-error' component={AppError} />
+                        <PluginLayout
+                          path='/coinview'
```

Comment:
I think we should prepend `/plugin` to anything in plugin layout please, since its totally different from rest of app.

change to `/plugins/coins-list`